release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -93,7 +93,7 @@ Release Team selection should happen in accordance with the [Release Team select
   - Enhancements tracking spreadsheet: `http://bit.ly/k8sXYY-enhancement-tracking`
   - Merged PRs with release notes: `http://bit.ly/k8sXYY-relnotes`
   - Use the same conventions for additional documents
-- Burndown meetings happen at 10AM Pacific Time, and you invite the and the Kubernetes Release calendar (`agst.us_b07popf7t4avmt4km7eq5tk5ao@group.calendar.google.com`) to them.
+- Burndown meetings happen at 10AM Pacific Time, and you invite the release team members and the Kubernetes Release calendar (`agst.us_b07popf7t4avmt4km7eq5tk5ao@group.calendar.google.com`) to them.
```

Comment:
We can also update this to mention only inviting the release calendar if that makes more sense.
```suggestion
- Burndown meetings happen at 10AM Pacific Time, and you invite the Kubernetes Release calendar (`agst.us_b07popf7t4avmt4km7eq5tk5ao@group.calendar.google.com`) to them.
```